lib/realtime/vehicle.ex

```diff
@@ -1,15 +1,30 @@
 defmodule Realtime.Vehicle do
+  alias Gtfs.{Direction, Route, Stop, Timepoint, Trip}
+
   @type current_status() :: :in_transit_to | :stopped_at
 
+  @type stop_timepoint_status() :: %{
+          stop_granularity: %{
+            current_status: current_status(),
```

Comment:
`current_status` is such a terrible name, and I'm a little mad at the GTFS-RT spec for using it. How about field names `stop_status` and `timepoint_status` instead?